src/ontodia/viewUtils/htmlToSvg.ts

```diff
@@ -0,0 +1,422 @@
+import { uniqueId } from 'lodash';
+
+export function isHTMLImageElement(e: Node): e is HTMLImageElement {
+    return (e instanceof HTMLImageElement);
```

Comment:
I think it's unnecessary to create these type guards with a trivial implementation. It's fine to just write `if (e instanceof HTMLImageElement) { ... }` without `isHTMLImageElement` wrapper.

Same for other `is*()` type guards here.